### PR TITLE
fix(ui5-combobox, ui5-multicombobox): center ValueState text

### PR DESCRIPTION
**Pull request by ndeshev**
ValueState text is now vertically centered in the Popover's header.

Issue: #2274

### CODE REVIEW

**Review by ilhan007 — CHANGES_REQUESTED** [review]
Because the ResponsivePopover.css styles take effect in input with suggestions, combobox, multi combobox, multi input, select and all date-time components, I think the value state text alignment of the text should be checked for them as well. 
(1) Check if  this is a bug only in the ComboBox
and 
(2) make sure the fix won't break the rest of the components.

**Comment by ndeshev** [comment]
The issue was also present in MultiComboBox, I added a fix for it and reverted back the previous value of the responsive popover's header CSS height param.

File: packages/main/src/themes/base/sizes-parameters.css [hunk 1]
```diff
@@ -69,9 +69,6 @@
 	/* TextArea */
 	--_ui5_textarea_padding: 0.5625rem 0.6875rem;
 
-	/* Responsive Popover */
-	--_ui5-responnsive_popover_header_height: 2.75rem;
```
Comment (ilhan007) [inline packages/main/src/themes/base/sizes-parameters.css:73]: Should we remove the cozy param? I think we should keep it and just fix the typo - responnsive -> responsive

**Review by ilhan007 — CHANGES_REQUESTED** [review]
It works for single line, but not for the valueStateMessage slot. I guess it is the line-height

Screenshot 2021-01-27 at 1 15 34 PM

**Review by ilhan007 — APPROVED** [review]

**Pull request merged**